docs/assembly.rst

```diff
@@ -263,8 +263,9 @@ Here is an assembly snippet you can use for allocating memory that follows the p
     }
 
 The first 64 bytes of memory can be used as "scratch space" for short-term
-allocation. The 32 bytes after the free memory pointer (i.e., starting at ``0x60``)
-are meant to be zero permanently and is used as the initial value for
+allocation. The free memory pointer is itself 32 bytes. The following 32 bytes
```

Comment:
> The free memory pointer is itself 32 bytes

While that is true, it's somewhat implicit as Yul variables are the same size as EVM words (32 bytes), there is nothing special here.

Reply:
The point of stating the free memory pointer size here was to allow new users to understand the pointer math that yields 0x80.

Actually this whole PR will be closed once you answer the above question(s). This section was changed a lot by another PR to address the Layout in Memory changes.

Reply:
So, since the other question was answered, should this PR still be reviewed or will you be closing it?